README.md

```
@@ -1,165 +1,279 @@
+| **Backend (Produção)** | [https://apae-gestao-escolar.onrender.com/](https://apae-gestao-escolar.onrender.com/) |
+| **API Docs (Produção)** | [https://apae-gestao-escolar.onrender.com/swagger-ui/index.html?urls.primaryName=Turmas#/Turmas/listarTodas](https://apae-gestao-escolar.onrender.com/swagger-ui/index.html?urls.primaryName=Turmas#/Turmas/listarTodas) |
 
-Prettier	        Formatador automático de código
+> ⚡ **Disponibilidade:** O serviço pode levar alguns segundos para iniciar em caso de inatividade (free tier).
 
-Git	            Controle de versão
+## 🚀 Tecnologias
 
-VS Code	        Editor de código recomendado
+### Frontend
+| Tecnologia | Descrição |
```

markdownlint-cli2 (0.21.0)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)